### PR TITLE
Jakarta Deployment 1.7 spec

### PR DESCRIPTION
**Pull request by m0mus**
## Specification PR template
When creating a specification project release review, create two PRs with the content divided as follows.

Include the following in PR#1:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat_x.y.pdf
- [x] Specification HTML in the form of wombat_x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [x] For a Release Review, a summary that a Compatible Implementation
      is complete, passes the TCK, and that the TCK includes sufficient
      coverage of the specification. The TCK users guide MUST include
      the instructions to run the compatible implementations used to
      validate the release.  Instructions MAY be by reference.
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://oss.sonatype.org/content/repositories/staging/jakarta/enterprise/deploy/jakarta.enterprise.deploy-api/1.7.2/
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/promoted/eclipse-jakartaeetck-8.0.0.zip
- [x] The URL of the compatibility certification request issue:
      https://github.com/eclipse-ee4j/enterprise-deployment/issues/41

Include the following in PR#2:
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory.

### CODE REVIEW

File: deployment/1.7/_index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+date: 2019-07-30T10:00:00+0100
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Deployment defines standard APIs that will enable any deployment tool that uses the
+deployment APIs to deploy any assembled application onto a Jakarta EE compatible platform.
+
+* [Jakarta Deployment 1.7 Specification Document](./deployment-spec-1.7.pdf) (PDF)
+* [Jakarta Deployment 1.7 Specification Document](./deployment-spec-1.7.html) (HTML)
+* [Jakarta Deployment 1.7 Javadoc](./apidocs)
+* Maven coordinates
```
Comment (starksm64) [inline deployment/1.7/_index.md]: Add link to platform TCK

**Comment by starksm64** [comment]
Remove deployment/1.7/images/jakarta_ee_logo_schooner_color_stacked_default.png

File: deployment/1.7/_index.md (hunk 1 above)
Comment (starksm64) [reply deployment/1.7/_index.md]: Remove timestamp when updating _index.md

**Comment by starksm64** [comment]
Make sure the correct platform TCK results/SHA is used for recent changes.

File: deployment/1.7/_index.md (hunk 1 above)
Comment (m0mus) [reply deployment/1.7/_index.md]: Fixed

**Comment by starksm64** [comment]
# Spec Review Checklist

1. Spec PR
  - [x] PR uses [template](https://github.com/jakartaee/specifications/blob/master/pull_request_template.md)
  - [x] Directory of form {spec}/x.y
  - [x] PDF of form {spec}-spec-x.y.pdf ("-spec" preferred but not required)
  - [x] HTML of form {spec}-spec-x.y.html ("-spec" preferred but not required)
  - [x] Index page {spec}/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
  - [x] Index page {spec}/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_index_template.md)
  - [x] No other files (e.g., no jakarta_ee_logo_schooner_color_stacked_default.png)
  - [x] Staging repository link of the form https://oss.sonatype.org/conte/repositories/staging/jakarta/{spec}/jakarta-{spec}-api/x.y.z/
  - [x] EFTL TCK link of the form http://download.eclipse.org/.../+.zip
  - [x] Compatibility certification link of the form https://github.com/eclipse-ee4j/{project}/#{issue}
  - [x] Second PR for just apidocs

2. _index.md
  - [x] Link to spec pdf
  - [x] Link to spec html
  - [x] Link to apidocs
  - [x] Link to final TCK download zip file of the form https://download.eclipse.org/jakartaee/{spec}/x.y/*{spec}-tck-x.y.z.zip
  - [x] Link to API jar file of the form https://search.maven.org/artifact/jakarta.{spec}/jakarta.{spec}-api/x.y.z/jar
  - [x] Name of and link to at least one Compatible Implementation

3. javadocs
  - [x] Footer contains Eclipse copyright and link to license
  - [x] ESFL license is included, usually as doc-files/speclicense.html
  - [x] no META-INF directory in PR

4. Spec PDF
  - [x] Correct spec title
  - [x] Version number of the form x.y, not x.y.z
  - [x] Correct Eclipse copyright line
  - [x] No DRAFT or SNAPSHOT

5. Spec HTML
  - [x] Same as PDF

6. TCK zip file
  - [ ] README file (optional for this release)
  - [x] EFTL license file, preferably named LICENSE.md
  - [x] User's Guide (or equivalent documentation)
  - [=x] How to test the Compatible Implementation(s) listed in _index.md above with the TCK (may be in UG)

7. TCK User's Guide (or equivalent documentation)
  - [x] Software requirements listed
  - [x] Installation and configuration described
  - [x] How to run tests
  - [x] Where to file challenges

8. Compatibility certification request
  - [x] Request follows [template](https://github.com/jakartaee/specification-committee/blob/master/compatibility-certification-request.md)
  - [x] SHA-256 fingerprint matches staged TCK zip file
847c80c9c80bea4682006f186292b68acdd0ce9b239d998856c3a379c3a7be50
  - [x] Request issue has `certification` label.

9. TCK results summary
  - [x] Page is hosted by Compatible Implementation project
  - [x] Includes all information from certification request
  - [x] Summary includes number of tests passed, failed, errors

**Comment by starksm64** [comment]
- [ ] The designated specification committee representative adds this final checklist to the main PR.
 - [ ] The specification committee representative merges the specification PRs.
 - [ ] The designated specification committee representative calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job.
 - [ ] The specification project member who created the staging release promotes the specification api jars to maven central.
 - [ ] The EMO updates the specification page with the ballot results.
This list goes on the committed spec index page.
 - [ ] The specification project team should go through the merged spec website page to verify all the links are valid.
 - [ ] The specification project team should approve the compatibility request and send an email to tck@eclipse.org for approval of the compatible implementation for trademark usage.
 - [ ] The specification project team should merge any final release branch as approriate for the branch management for the project.

**Comment by kwsutter** [comment]
I just unchecked the box for the maven promotion step since I couldn't find it in maven:
https://mvnrepository.com/artifact/jakarta.enterprise.deploy/jakarta.enterprise.deploy-api

**Comment by starksm64** [comment]
Ok, I had followed the https://search.maven.org/artifact/jakarta.enterprise.deployment/jakarta.enterprise.deploy-api/1.7.2/jar link, and that shows something, but I guess it is invalid as when I hit the browse button, that did return a 404.

**Comment by starksm64** [comment]
@m0mus Can you release the api jars to maven central.

**Comment by m0mus** [comment]
It's fine now: https://search.maven.org/artifact/jakarta.enterprise.deploy/jakarta.enterprise.deploy-api/1.7.2/jar

**Comment by m0mus** [comment]
Submitted PR https://github.com/jakartaee/specifications/pull/111 fixing Maven Central and TCK links.

**Comment by m0mus** [comment]
- [x] The specification project member who created the staging release promotes the specification api jars to maven central.
- [x] The specification project team should go through the merged spec website page to verify all the links are valid.
- [x] The specification project team should approve the compatibility request and send an email to tck@eclipse.org for approval of the compatible implementation for trademark usage.
- [x] The specification project team should merge any final release branch as appropriate for the branch management for the project.

**Pull request merged**